docs/data/material/components/text-fields/text-fields.md

```diff
@@ -70,6 +70,12 @@ For instance, you can use an icon button to hide or reveal the password.
 
 {{"demo": "InputAdornments.js"}}
 
+### Visible on shrink
+
+Use the `sx` prop to control the suffix's opacity based on the shrink attribute. The suffix will be visible when the label is shrunk.
```

Comment:
Saying "Use the `sx` prop" might confuse users to think this can only be achieved via the `sx` prop. Can we frame it differently? Instead of "`sx` prop" say "custom styles" or something similar?

Reply:
Good point.

Reply:
Updated.